modules/ossm-update-app-sidecar.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-update-app-sidecar_{context}"]
+= Updating your application pods
+
+If you selected the Automatic Approval Strategy when you were installing your Operators, they update automatically. If you're updating {ProductName} from a previous version, you must update the sidecars on your application pods to the most recent version. 
```

Comment:
is it better to say "then the operators update automatically"? The sidecars have to be updated regardless.

Reply:
done

Reply:
@brian-avery I rewrote that first sentence with some input from Kevin.